app/controllers/hosts_controller.rb

```diff
@@ -662,7 +666,7 @@ def resource_base
 
   define_action_permission [
     'clone', 'externalNodes', 'overview', 'bmc', 'vm', 'runtime', 'resources', 'templates', 'nics',
-    'pxe_config', 'storeconfig_klasses', 'active', 'errors', 'out_of_sync', 'pending', 'disabled', 'get_power_state', 'preview_host_collection'
+    'pxe_config', 'storeconfig_klasses', 'active', 'errors', 'out_of_sync', 'pending', 'disabled', 'get_power_state', 'preview_host_collection', 'build_errors'
```

Comment:
Is this maybe worth an extra permission? Can the build logs contain sensitive information you don't want every user with view permissions to see?

Reply:
I tend to think view permission fine, I assume that this gives you permission to actually view cfg mgmt parameters anyway and log is just a result of that?